src/metabase/public_settings/premium_features.clj

```diff
@@ -208,27 +214,25 @@
   (cond (mc/validate [:re RemoteCheckedToken] token)
         ;; attempt to query the metastore API about the status of this token. If the request doesn't complete in a
         ;; reasonable amount of time throw a timeout exception
-        (do
-          (log/infof "Checking with the MetaStore to see whether token '%s' is valid..." (u.str/mask token))
-          (let [site-uuid (setting/get :site-uuid-for-premium-features-token-checks)]
-            (try (fetch-token-and-parse-body token token-check-url site-uuid)
-                 (catch Exception e1
-                   ;; Unwrap exception from inside the future
-                   (log/errorf e1 "Error fetching token status from %s:" token-check-url)
-                   ;; Try the fallback URL, which was the default URL prior to 45.2
-                   (try (fetch-token-and-parse-body token store-url site-uuid)
-                        ;; if there was an error fetching the token from both the normal and fallback URLs, log the
-                        ;; first error and return a generic message about the token being invalid. This message
-                        ;; will get displayed in the Settings page in the admin panel so we do not want something
-                        ;; complicated
-                        (catch Exception e2
-                          (log/errorf e2 "Error fetching token status from %s:" store-url)
-                          (let [body (u/ignore-exceptions (some-> (ex-data e1) :body (json/parse-string keyword)))]
-                            (or
-                             body
-                             {:valid         false
-                              :status        (tru "Unable to validate token")
-                              :error-details (.getMessage e1)}))))))))
+        (let [site-uuid (setting/get :site-uuid-for-premium-features-token-checks)]
+          (try (fetch-token-and-parse-body token token-check-url site-uuid)
+               (catch Exception e1
+                 ;; Unwrap exception from inside the future
```

Comment:
I think this was my fault from my previous refactor and definitely not a blocker but I just noticed that this comment is outdated, we're no longer unwrapping anything.